samples/README.md

```diff
@@ -0,0 +1,25 @@
+[//]: # "This README.md file is auto-generated, all changes to this file will be lost."
+[//]: # "To regenerate it, use `python -m synthtool`."
+
+## Python Samples for Cloud Bigtable
+
+This directory contains samples for Cloud Bigtable, which may be used as a refererence for how to use Cloud Bigtable. 
+
+## Additional Information
+
+You can read the documentation for more details on API usage and use GitHub
+to [browse the source][source] and [report issues][issues].
+
+### Contributing
+For [contributing guidelines][contrib_guide], the [Python style guide][py_style], and more information on prerequisite steps to contribute, view the source code at <a href="https://github.com/googleapis/python-bigtable">googleapis/python-bigtable</a>.
+
+[authentication]: https://cloud.google.com/docs/authentication/getting-started
+[enable_billing]:https://cloud.google.com/apis/docs/getting-started#enabling_billing
+[client_library_python]: https://googlecloudplatform.github.io/google-cloud-python/
+[source]: https://github.com/GoogleCloudPlatform/google-cloud-python
```

Comment:
this link (and everywhere else) should be googleapis/python-bigtable

Reply:
Okay, making changes to the synthtool template. This will link will be replaced with a link to `https://github.com/{{ metadata['repo']['repo'] }}`